tox.ini

```diff
@@ -1,10 +1,10 @@
 [tox]
-envlist = py27, py35, py36, pypy, lint, packaging, docs
+envlist = py27, py35, py36, py37, pypy, lint, packaging, docs
 
 [testenv]
 deps =
-    pytest==3.2.1
-    pytest-cov==2.5.1
+    pytest
+    pytest-cov
```

Comment:
I think I like the approach in hyper-h2 more - with a dedicated `test_requirements.txt` to keep track of dev-only packages and their version dependencies...

Reply:
https://github.com/python-hyper/hyper-h2/blob/master/tox.ini#L5 and https://github.com/python-hyper/hyper-h2/blob/master/test_requirements.txt